Explorable-Explanations.md

```diff
@@ -1,11 +1,17 @@
-Explorable explanations represent a new form of interactive media, empowering readers to actively engage with and explore complex ideas and concepts.  They function as "reactive documents," allowing readers to manipulate the author's assumptions and analyses, directly observing the resulting consequences.
+Explorable explanations represent a new form of interactive media, empowering readers to actively engage with and explore complex ideas and concepts.
 
-The core idea is to move beyond passive reading and create documents designed for interaction.  The goal is to provide a more engaging and interactive experience, particularly for understanding complex logic and concepts, such as those within the Privacy Sandbox APIs.
+They function as "reactive documents," allowing readers to manipulate the author's assumptions and analyses, directly observing the resulting consequences.
 
-By interacting with these explanations, readers can explore the impact of different assumptions, for example, joining various interest groups.  This allows for a deeper understanding of processes like auctions and bidding.
+### How PSAT uses Explorable Explanations
+The PSAT extension contains two explorable explanations for the Topics API and the Protected Audience API. It helps users view and interact with them and check how they react at each step.
 
-In summary, explorable explanations are a powerful new interactive medium that can significantly improve comprehension of complex topics, especially those related to the Privacy Sandbox APIs. They offer a valuable tool for both developers and advertisers, enhancing accessibility and engagement with intricate concepts.
+#### [Topics API](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/Topics-EE)
+The Explorable Explanations provide a visual representation of how the Topics API assigns topics to users based on their Browse behavior. Users can visit different nodes, and as they interact, they are assigned to different topics.
+
+#### [Protected Audience API](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/Protected-Audience-EE)
+The demo is structured to simulate a user's online journey, showcasing the interactions between different types of websites (nodes) and the dynamic processes involved in ad auctions. As a user visits an advertiser node, they are added to interest groups. When they visit a publisher's website, these interest groups are accessed, and an auction happens.
 
 ## Resources
-- [Wikipedia](https://en.wikipedia.org/wiki/Explorable_explanations)
+- [Wikipedia](https://en.wikipedia.org/wiki/Explorable_explanation)
 - [Explorable Explanations](https://explorabl.es/)
+- [Brett Victor](https://worrydream.com/ExplorableExplanations/)
```

Comment:
Lets remove this article as well, only Wikipedia link should suffice

Reply:
Removed, kept only wikipedia link